src/posthog-core.js

```diff
@@ -131,6 +132,8 @@ var create_mplib = function (token, config, name) {
     instance.sessionRecording = new SessionRecording(instance)
     instance.sessionRecording.startRecordingIfEnabled()
 
+    decide.init(instance)
```

Comment:
So this refactoring has a non-intuitive side-effect which might be considered a regression - we now always call /decide. 

Previously decide was not called when autocapture was off - making the script a lot more light-weight if nothing was turned on.

Reply:
Ack you're right. Well definitely something worth changing would be to not call the `/decide` endpoint if session recording and autocapture are both disabled. Perhaps if we want to avoid the extra request to `/decide` we need to introduce a different param that enables session recording even if autocapture is off? Thoughts?

Reply:
Per sync conversation, we'll keep the new approach as `/decide` is also required for other stuff like compression and feature flags, and adding an advanced config option to fully disable this request.